element_lab/project.py

```diff
@@ -0,0 +1,196 @@
+
+
+@schema
+class ProjectPersonnel(dj.Manual):
+    """List of individuals involved in a project
+
+    Attributes:
+        Project (foreign key): Project key
+        User (foreign key): User key
+    """
+
+    definition = """# List of individuals involved in a project
+    -> Project
+    -> User
+    """
+
+
+@schema
+class ProjectKeywords(dj.Manual):
+    """Project keywords, exported dataset meta info
+
+    Attributes:
+        Project (foreign key): Project key
+        keyword ( varchar(32) ): Keywords describing the project
+    """
+
+    definition = """
+    # Project keywords, exported dataset meta info
+    -> Project
+    keyword: varchar(32) # Keywords describing the project
+    """
+
+
+@schema
+class ProjectPublication(dj.Manual):
+    """Project's resulting publications
+
+    Attributes:
+        Project (foreign key): Project key
+        publication ( varchar(255) ): Publication name or citation
+    """
+
+    definition = """
+    # Project's resulting publications
+    -> Project
+    publication: varchar(255)  # Publication name or citation
+
+    """
+
+
+@schema
+class ProjectSourceCode(dj.Manual):
+    """URL to source code for replication
+
+    Attributes:
+        Project (foreign key): Project key
+        repository_url ( varchar(255) ): Link to code repository
+        repository_name ( varchar(32), optional): Name of code repository
+    """
+
+    definition = """# URL to source code for replication
+    -> Project
+    repository_url     : varchar(255)  # Link to code repository
+
+    ---
+    repository_name='' : varchar(32)   # Name of code repository
+    """
+
+
+@schema
+class Study(dj.Manual):
+    """A set of experiments designed to address a specific aim
+
+    Attributes:
+        Project (foreign key): Project key
+        study ( varchar(24) ): Abbreviated study name, e.g., 'Aim 1'
+        study_name ( varchar(128) ): Full study name, e.g., 'perceptual response tasks',
+            or 'Aim 1'
+        study_description ( varchar(1024), optional): Description of study goals,
+            objectives, and/or methods
+    """
+
+    definition = """# A set of experiments designed to address a specific aim
+    -> Project
+    study                : varchar(24)    # Abbreviated study name, e.g., 'Aim 1'
+    ---
+    study_name           : varchar(128)   # Full name, e.g., 'perceptual response tasks'
+    study_description='' : varchar(1024)  # Description of study goals, objectives, and/or methods
+    """
+
+    class Protocol(dj.Part):
+        """Info about institutional approval (e.g., IACUC, IRB, etc.)
+
+        Attributes:
+            Study (foreign key): Study key
+            Protocol (foreign key): Protocol key
+        """
+
+        definition = """# Info about institutional approval (e.g., IACUC, IRB, etc.)
+        -> master
+        -> Protocol
+        """
+
+
+@schema
+class Experiment(dj.Manual):
+    """Experimental tasks/protocols and their associated lab and study

```

Comment:
```suggestion
    """Experimental tasks and their associated lab, study, and protocol
```

Reply:
Since `protocol` is already used in this schema to describe something different.